wagtail/project_template/project_name/settings/base.py

```diff
@@ -52,7 +52,7 @@
     'django.contrib.staticfiles',
 ]
 
-MIDDLEWARE_CLASSES = [
+MIDDLEWARE = [
```

Comment:
It seems to me that this project template will not work for Wagtail 1.6.* Django 1.9.* or 1.8.*?